src/PkgDeps.jl

```diff
@@ -166,18 +166,22 @@ end
 
 
 """
-    users(pkg_name::String; pkg_registry_name::String="$GENERAL_REGISTRY")
```

Comment:
I'm not sure how to handle these situations here. Since we pass in `kwargs...`, and don't specify the name of them in this function but you need more knowledge of how this function works by calling others.

Reply:
yeah, i'd prefer `pkg_registry_name` to be a postional arg with a default, and all methods have the same keywords
https://github.com/JuliaEcosystem/PkgDeps.jl/pull/20#discussion_r617702305

Reply:
oh now i see you can also pass `depot=...`

I wonder if we can improve design there? e.g. just pass a `RegistryInstance` rather than `pkg_registry_name::String, depot::String`?

Reply:
I kept them both at strings as you won't need to to make an extra call just to query based off of a `pkg_name`.

Reply:
we can have that too (i've made both possible in #21)